website/docs/patterns/multi-select-patterns/partials/specifications/specifications.md

```diff
@@ -0,0 +1,43 @@
+## Anatomy
+
+![Multi-select anatomy](/assets/patterns/multi-select-patterns/selection-anatomy.png)
```

Comment:
In other parts the "Global selection options" is referred to as "Bulk selection", I think you have to update the image.